js/components/stories/Breakpoints.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+
+const React = require("react");
+const { DOM: dom, createElement } = React;
+const { storiesOf } = require("@kadira/storybook");
+const configureStore = require("../../create-store");
+const { combineReducers } = require("redux");
+const reducers = require("../../reducers");
+const { Provider } = require("react-redux");
+const { fromJS } = require("immutable");
+
+const Breakpoints = React.createFactory(require("../Breakpoints"));
+
+const fixtures = {
+  sources: {
+    "fooSourceActor": {
+      actor: "fooSourceActor",
+      url: "http://example.com/foo.js"
+    },
+    "barSourceActor": {
+      actor: "barSourceActor",
+      url: "http://example.com/bar.js"
+    }
+  },
+  breakpoints: {
+    "fooBreakpointActor": {
+      actor: "fooBreakpointActor",
+      location: {
+        actor: "fooSourceActor",
+        line: 16
+      },
+    },
+    "barBreakpointActor": {
+      actor: "barBreakpointActor",
+      location: {
+        actor: "barSourceActor",
+        line: 18
+      },
+    }
+  }
+};
+
```

Comment:
This fixture data hopefully can move to its own file soon and be re-used by component unit tests